Airspace: add Australia HG/PG SiteGuide

elgandoz opened the pull request:
This airspace adds Recommended/Restricted/Prohibited LZs and dangers/obstacles (like power lines), for the most active flying sites for hang gliders and paragliders in Australia.
The airspace file is managed by several clubs members and it is continuously updated, due the continuous changes and agreements with the land owners.

Source: https://siteguide.org.au/

lordfolken commented:
Hey, thanks for your contribution and making XCSoar that much more useful!

Since this isn't an 'official' airspace of a country, it would suggest to put it into https://github.com/XCSoar/xcsoar-data-repository/blob/master/data/airspace-special.json instead of airspace.json
This is purley organizational, and it will show up in the regular airspace list alphabetically sorted.

elgandoz commented:
@lordfolken, thanks for the reccomendations (I should have read the README!).
This is now ready for review.

elgandoz commented:
@lordfolken, Thanks for the merge, although it doesn't work downloading correctly.
I reckon it is due to the `name` not being an actual *.txt filename.
My very first commit was using a filename, but in the amend I converted it to a _label_, deceived by the others available in the file (Czech Wave Airspace, Polish Wave Airspace - those are not working too).
I fixed my commit, are you be able to force push again?
I'm sorry for the inconvenience, and thanks for the help.

lordfolken commented:
Hmm. Thanks for catching the other two errors. 
I'm also a bit unsure about the '/' in the filename. This is a subdirectory on *nix (kobo,android,openvario,unix) like systems.
